### ISSUE TITLE
can't pipe output

### ISSUE THREAD

**Issue by timruffles** [julien-f/csv2json#2]
makes it rather less useful as a cli tool

e.g 

``` sh
$ pbpaste csv2json | jq '.'
✖ Property 'read' of object #<Socket> is not a function
TypeError: Property 'read' of object #<Socket> is not a function
    at Socket.<anonymous> (_stream_readable.js:763:45)
    at Socket.emit (events.js:92:17)
    at emitDataEvents (_stream_readable.js:789:10)
    at Socket.Readable.on (_stream_readable.js:710:5)
    at proxyStream (/usr/local/lib/node_modules/csv2json/node_mod
```

**Comment by julien-f** [comment]
Can you check the latest GitHub version?

If it works for you I'll release it.

**Comment by julien-f** [comment]
@timruffles ping?

**Comment by timruffles** [comment]
Yup looks good to me.

**Comment by julien-f** [comment]
Released as 1.0.1.
